flink-table/flink-table-planner/src/test/resources/restore-tests/stream-exec-group-window-aggregate_1/group-tumble-window-proc-time/plan/group-tumble-window-proc-time.json

```diff
@@ -306,51 +374,9 @@
         }
       },
       "isTimeWindow" : true,
-      "size" : "PT15M"
+      "size" : "PT5S"
     },
-    "namedWindowProperties" : [ {
-      "name" : "w$start",
-      "property" : {
-        "kind" : "WindowStart",
-        "reference" : {
-          "name" : "w$",
-          "type" : {
-            "type" : "TIMESTAMP_WITH_LOCAL_TIME_ZONE",
-            "nullable" : false,
-            "precision" : 3,
-            "kind" : "PROCTIME"
-          }
-        }
-      }
-    }, {
-      "name" : "w$end",
-      "property" : {
-        "kind" : "WindowEnd",
-        "reference" : {
-          "name" : "w$",
-          "type" : {
-            "type" : "TIMESTAMP_WITH_LOCAL_TIME_ZONE",
-            "nullable" : false,
-            "precision" : 3,
-            "kind" : "PROCTIME"
-          }
-        }
-      }
-    }, {
-      "name" : "w$proctime",
-      "property" : {
-        "kind" : "Proctime",
-        "reference" : {
-          "name" : "w$",
-          "type" : {
-            "type" : "TIMESTAMP_WITH_LOCAL_TIME_ZONE",
-            "nullable" : false,
-            "precision" : 3,
-            "kind" : "PROCTIME"
-          }
-        }
-      }
-    } ],
+    "namedWindowProperties" : [ ],
```

Comment:
Yes, because the query is not projecting window_start, window_end for proc time tests in the Sink schema. I only added them for tumble-window-event-time test and therefore its populated in that plan.

Reply:
Basically, the WindowPropertiesRules doesn't get fired

Reply:
ok, I see

Reply:
Yes